client/submitAssignment.js

```diff
@@ -20,7 +24,11 @@ window.onload = function() {
     });
 };
 
-// Uploads the form data to server.
+/**
+ * Validates assignment tarball file and sends a POST request to server
+ * for uploading the tarball and other assignment related details.
```

Comment:
What other assignment related details are we talking about here?

Reply:
No, its only the tarball. I removed the addtional comment.